### ISSUE TITLE
Error in validateExperiments(df): stage and libtype must be colnames in df!

### ISSUE THREAD

**Issue by SuhasSrinivasan** [Roleren/ORFik#157]
Hi,

Thank you for making this helpful package and hope this can be a unifying tool!

I am currently following the Ribo-seq vignette to load some Ribo-seq data
https://bioconductor.org/packages/release/bioc/vignettes/ORFik/inst/doc/Ribo-seq_pipeline.html

However, there is an issue at this step
`list.experiments()` 

```
Error: BiocParallel errors
  2 remote errors, element index: 1, 2
  0 unevaluated and other errors
  first remote error:
Error in validateExperiments(df): stage and libtype must be colnames in df!
```

I am not sure about the BiocParallel error, but traced the validateExperiment to here:
https://github.com/Roleren/ORFik/blob/49a3d30beb8e9d2bfd8df1db4ee9d4b6b9cf42cc/R/experiment.R#L88 

Help would be greatly appreciated!

**Comment by SuhasSrinivasan** [comment]
Following the vignette, `create.experiment()` was not supplied with `libtype` and `stage`, i.e., default `auto`.

After supplying values, e.g., `libtype = RFP` and `stage = CellLine` verified that the CSV file written to directory ORFik_experiments contains the changes.

However, the error still persists.

**Comment by SuhasSrinivasan** [comment]
Copying the CSV file so that it can be helpful.

[colo_riboseq_Ribo-seq.csv](https://github.com/Roleren/ORFik/files/14567273/colo_riboseq_Ribo-seq.csv)

**Comment by Roleren** [comment]
This error was related to pairedEndBam flag and is now fixed in #159
